pages/pipelines/command_step.md

```diff
@@ -214,7 +214,7 @@ To prevent jobs from consuming too many job minutes or running forever, you can
 
 Specific timeouts take precedence over more general ones — a step level timeout takes precedence over a pipeline timeout, which in turn takes precedence over an organization default.
 
-Maximum timeouts are applied to any command step without a timeout or with a timeout greater than the maximum timeout.
+Maximum timeouts are applied to any command step without a timeout, default timeout not set in pipeline settings or with a timeout greater than the maximum timeout.
```

Comment:
Style: Use a serial comma.

```suggestion
Maximum timeouts are applied to any command step without a timeout, default timeout not set in pipeline settings, or with a timeout greater than the maximum timeout.
```

Reply:
Actually, I think the issue is how hard the sentence is to parse. How about splitting it into a bulleted list? Thinking:

```suggestion
Maximum timeouts are applied to command steps in the following situations:

- No timeout attribute is set on the step.
- No default timeout is set in the pipeline settings.
- When the timeout set is greater than the maximum timeout.
```

Reply:
@mbelton-buildkite I like this bullet list approach as it is easy to read and understand. I updated the PR with this approach